test/generate-build-solution/build.gradle.kts

```diff
@@ -0,0 +1,38 @@
+group = "test.de.itemis.mps.gradle.generate"
+version = "1.0-SNAPSHOT"
+
+plugins {
+    id("generate-models") version "1.0-SNAPSHOT"
```

Comment:
This should now be 1.1-SNAPSHOT. @arimer is this maybe the problem that was preventing you from testing the plugin?